core/src/main/java/org/infinispan/encoding/DataConversion.java

```diff
@@ -43,152 +38,155 @@
 @Scope(Scopes.NONE)
 public final class DataConversion {
 
+   /**
+    * @deprecated Since 11.0. To be removed in 14.0, with no replacement.
```

Comment:
14.0? Is this 2 years from now?  :)

Reply:
I hope it's going to be sooner than 2 years, but it's the example Tristan gave in the removal policy discussion.